### ISSUE TITLE
ModuleNotFoundError: No module named 'rtmlib.tools.solution.utils'

### ISSUE THREAD

**Issue by qingzew** [Tau-J/rtmlib#11]
in release 0.0.8, via ```pip install rtmlib -i https://pypi.org/simple```

**Comment by Tau-J** [comment]
Hi @qingzew , I cannot reproduce this error. Could you provide more information (things like the code you run)?

**Comment by qingzew** [comment]
```
import cv2

from rtmlib import Wholebody, draw_skeleton

device = 'cpu'  # cpu, cuda
backend = 'onnxruntime'  # opencv, onnxruntime, openvino

openpose_skeleton = False  # True for openpose-style, False for mmpose-style

wholebody = Wholebody(to_openpose=openpose_skeleton,
                      mode='balanced',  # 'performance', 'lightweight', 'balanced'. Default: 'balanced'
                                            backend=backend, device=device)

img = cv2.imread('./videoframe_0.png')
keypoints, scores = wholebody(img)
img_show = draw_skeleton(img, keypoints, scores, kpt_thr=0.5)
cv2.imwrite('img1.jpg', img_show)
print(keypoints.shape)

```

Traceback (most recent call last):
  File "/data1/demo.py", line 3, in <module>
    from rtmlib import Wholebody, draw_skeleton
  File "/usr/local/lib/python3.10/dist-packages/rtmlib/__init__.py", line 1, in <module>
    from .tools import (RTMO, YOLOX, Body, Hand, PoseTracker, RTMDet, RTMPose,
  File "/usr/local/lib/python3.10/dist-packages/rtmlib/tools/__init__.py", line 3, in <module>
    from .solution import Body, Hand, PoseTracker, Wholebody
  File "/usr/local/lib/python3.10/dist-packages/rtmlib/tools/solution/__init__.py", line 4, in <module>
    from .wholebody import Wholebody
  File "/usr/local/lib/python3.10/dist-packages/rtmlib/tools/solution/wholebody.py", line 48, in <module>
    from .utils.types import BodyResult, Keypoint, PoseResult
ModuleNotFoundError: No module named 'rtmlib.tools.solution.utils'

python version
```
Python 3.10.12 (main, Jun 11 2023, 05:26:28) [GCC 11.4.0] on linux
```
system info
```
Linux version 3.10.0-693.el7.x86_64 (builder@kbuilder.dev.centos.org) (gcc version 4.8.5 20150623 (Red Hat 4.8.5-16) (GCC) ) #1 SMP Tue Aug 22 21:09:27 UTC 2017
```

problem
```
utils/types.py not in the installed dir
```

**Comment by Tau-J** [comment]
That's weird. Perhaps you can try reinstalling again, or install via clone this repo and run `pip install -e .`

**Comment by mprib** [comment]
I will chime in here and say that I ran into the same issue with the pip install , ultimately using the editable install as suggested above.

I think that the version on pip may be out of date. There is no utils within rtmlib.tools.solution that is packaged in PyPI:

![image](https://github.com/Tau-J/rtmlib/assets/31831778/af56df62-a7c5-4fec-9d19-0d13e821a093)

**Comment by Tau-J** [comment]
> I will chime in here and say that I ran into the same issue with the pip install , ultimately using the editable install as suggested above.
> 
> I think that the version on pip may be out of date. There is no utils within rtmlib.tools.solution that is packaged in PyPI:
> 
> ![image](https://private-user-images.githubusercontent.com/31831778/302997836-af56df62-a7c5-4fec-9d19-0d13e821a093.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDczMjMwMjUsIm5iZiI6MTcwNzMyMjcyNSwicGF0aCI6Ii8zMTgzMTc3OC8zMDI5OTc4MzYtYWY1NmRmNjItYTdjNS00ZmVjLTlkMTktMGQxM2U4MjFhMDkzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAyMDclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMjA3VDE2MTg0NVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTZmMWEzMTgyNWI3MDNjZmQ4NDhhMjZiMmZlYzc0ODI4ZjU3ZDhjOWM0YjI3MTJmMzlkYjE2Zjg5MDk1NDA1MjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GHbHCApHzLa6XcvBVBTs2d76DI3SuP588EHSs0gPQHI)

Thanks for pointing this out. I'll update the pipy soon.

**Comment by qingzew** [comment]
still missing rtmlib.tools.solution.utils, release 0.0.9

**Comment by Tau-J** [comment]
> still missing rtmlib.tools.solution.utils, release 0.0.9

This is the source code downloaded from https://github.com/Tau-J/rtmlib/releases/tag/0.0.9

![image](https://github.com/Tau-J/rtmlib/assets/13503330/8e60473a-e260-47bc-bd9d-0d51a5ee7952)

**Comment by Tau-J** [comment]
> still missing rtmlib.tools.solution.utils, release 0.0.9

I have released 0.0.10 to fix this bug, could you please help to give me any feedback? Thanks in advance.

**Comment by qingzew** [comment]
another problem
```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.10/dist-packages/rtmlib/__init__.py", line 3, in <module>
    from .visualization.draw import draw_bbox, draw_skeleton
ModuleNotFoundError: No module named 'rtmlib.visualization'

```
